File: THE LANGUAGE/00/BuiltIn/DataStructures/Objects/object.cpp
#include <bits/stdc++.h>


using namespace std;

struct person{
    string name;
    int age;
    void hello(){
        cout << "Hello, I'm Marouane" << endl;
    }
};


int main(void){

    person maroci = {"Marouane", 22};
    maroci.hello();

    return 0;
}

File: THE LANGUAGE/00/BuiltIn/DataStructures/Objects/object.rs
struct Person {
    name: String,
    age: i32,
}

impl Person {
    fn hello(&self) {
        println!("Hello, I'm {} and I'm {}", self.name, self.age);
    }
}

fn main() {
    let maroci = Person {
        name: String::from("Marouane"),
        age: 22,
    };

    maroci.hello();
}
